app/src/pages/eventspage/EventsEditPage.tsx

```diff
@@ -290,20 +351,29 @@ import { Dialog, DialogClose, DialogContent, DialogDescription, DialogFooter, Di
                                 <div className="grid gap-6">
                                     <div className="grid gap-3">
                                         <Label htmlFor="food">Food</Label>
-                                        <Select>
+                                        <Select
+                                            defaultValue={eventLocal.food}
+                                            onValueChange={(value)=>{
+                                                setEventLocal((prevEventLocal: any) => ({
+                                                    ...prevEventLocal,
+                                                    food: value,
+                                                }))
+                                                console.log(eventLocal)
+                                            }}
+                                        >
                                             <SelectTrigger id="food" aria-label="Select food">
                                                 <SelectValue placeholder="Select food" />
                                             </SelectTrigger>
                                             <SelectContent>
-                                                <SelectItem value="chick">Chick Fil-A</SelectItem>
-                                                <SelectItem value="chip">Chipotle</SelectItem>
-                                                <SelectItem value="panera">Panera</SelectItem>
-                                                <SelectItem value="mikes">Jersey Mike's</SelectItem>
-                                                <SelectItem value="panda">Panda Express</SelectItem>
-                                                <SelectItem value="canes">Raising Canes</SelectItem>
-                                                <SelectItem value="snacks">Snacks</SelectItem>
-                                                <SelectItem value="other">Other</SelectItem>
-                                                <SelectItem value="none">None</SelectItem>
+                                                <SelectItem value="Chick Fil-A">Chick Fil-A</SelectItem>
```

Comment:
suggestion: these and other list-type enums can be put on a separate file and then you can read them from there.

it's easier to edit that way.